templates/languages/python/rules/testing.md

```
@@ -1,300 +1,300 @@
+
+[tool.coverage.report]
+precision = 2
+show_missing = true
+skip_covered = true
+```
+
+## Test Structure
+
+### File Naming
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)